src/assets/index.ts

```diff
@@ -0,0 +1,8 @@
+export const images = {
+    back: require('../../src/assets/back_white_24dp.png'),
```

Comment:
For easy readability, could you sort these alphabetically? This will make it easier later on for others to find the name of an image they are looking for. (There's a feature in vs code to sort lines)

Reply:
done